### PR TITLE
Move IngressFromManifest/IngressToManifest to ingress e2e fw

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

In e2e test framework, IngressFromManifest() and IngressToManifest are called in ingress e2e test
framework only. In addition, the functions work for ingress as tests
which call the functions do. So this moves the function into ingress
e2e test framework to make the dependency simple.

**Which issue(s) this PR fixes**:

Ref: #81232

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/ingress/ingress_utils_test.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```
Comment (oomichi) [inline test/e2e/framework/ingress/ingress_utils_test.go]: I don't think we need to update here.

Comment (gavinfish) [reply test/e2e/framework/ingress/ingress_utils_test.go]: done

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**